apps/src/StudioApp.js

```diff
@@ -2140,6 +2147,18 @@ StudioApp.prototype.configureDom = function (config) {
       }
     });
   }
+  // Sends a Statsig event when the Run button is pressed by a signed out user
+  // This is related to the Create Account Button A/B Test; see Jira ticket:
+  // https://codedotorg.atlassian.net/browse/ACQ-1938
+  if (runButton && isSignedOut && isStandaloneProject) {
+    dom.addClickTouchEvent(runButton, () => {
+      analyticsReporter.sendEvent(
+        EVENTS.RUN_BUTTON_PRESSED_SIGNED_OUT,
```

Comment:
Verified locally that doesn't show up on the Run button in a level (Pre-Express and All the things)
<img width="422" alt="Screenshot 2024-07-10 at 10 46 07 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/af35aa33-5854-49cf-aa9d-29f025a23b92">